docker/experian-crosscore/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM outofcoffee/imposter
```

I agree with SonarCloud on this, good to specify a version:
```suggestion
FROM outofcoffee/imposter:4.2.0
```